content/zh-cn/stateful-apps.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 有状态应用
+status: Completed
+category: 概念
+tags: ["基本原理", "", ""]
+---
+
+## 是什么
+
+当我们说到有状态（和[无状态](/zh-cn/stateless-apps/)）应用时，
```

Comment:
There is no `stateless-apps` under `/zh-cn`

Reply:
check it again, please. )

<img width="270" alt="image" src="https://user-images.githubusercontent.com/64075563/202829176-3c451fb9-a101-408b-a47a-41aefb87968c.png">

Reply:
Yes currently. PR #1472 has included the file of stateless-apps.